.github/workflows/release.yml

```diff
@@ -0,0 +1,41 @@
+name: Docker Image CI
+
+on:
+  push:
+    branches: 
+      - "*"
+  pull_request:
+    branches:
+      - "*"
+
+
+#on:
+  #release:
+    #types: [published]
+
+jobs:
+
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Build the Docker image
+      run: |
+         docker build . --file Dockerfile --rm -t "quay.io/razee/remoteresource:${GITHUB_SHA}"
+         docker images
+         ./build/process-template.sh kubernetes/RemoteResource/resource.yaml >/tmp/resource.yaml
+        
+  publish-npm:
```

Comment:
this should not run during PRs or pushes. This should only run for tagged releases